backstory/src/main/java/com/google/sps/images/VisionImagesManager.java

```diff
@@ -55,52 +57,56 @@ public VisionImagesManager(ImageAnnotatorClient imageAnnotatorClient) {
   @Override
   public List<AnnotatedImage> createAnnotatedImagesFromImagesAsByteArrays(
       List<byte[]> imagesAsByteArrays) throws IOException {
-    List<AnnotatedImage> annotatedImages = new ArrayList<>();
+    List<AnnotateImageRequest> requests = new ArrayList<AnnotateImageRequest>();
```

Comment:
Can this be declared below, just above where it's used? Declaring close to usage helps:
1. limit visibility to only code that needs it,
2. helps the reader associate related lines of code

Reply:
Thanks for the suggestion! I'll move it lower.

Reply:
Done